### DISCUSSION TITLE
当前账号需要短信验证码，请尝试修改UA或设备ID

### DISCUSSION THREAD

**Discussion by SeventeenOvO** [0-8-4/miui-auto-tasks#297] [问与答｜Q&A]
请大家帮忙看看这个是哪里的问题 青龙面板
![1](https://github.com/user-attachments/assets/00bfb4f3-016e-45b0-bf71-c8a7101d1abd)

**Comment by SeventeenOvO** [comment]
去服务器用浏览器登录了一下  还是这样

**Comment by TardisXE** [comment]
可能是帐号或服务器IP段被小米风控，试下在自家电脑运行一下

**Comment by quiz22** [comment]
你解决了吗？我刚刚部署也是这样

**Comment by quiz22** [comment]
过几天就行了。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2025年10月16日(周四) 中午11:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [0-8-4/miui-auto-tasks] 当前账号需要短信验证码，请尝试修改UA或设备ID (Discussion #297)




 
我在自家电脑也这样
